pandas/tests/test_frame.py

```diff
@@ -8881,6 +8881,16 @@ def test_drop(self):
         df.drop(labels=df[df.b>0].index, inplace=True)
         assert_frame_equal(df,expected)
 
+        # multi-index issue
+        # origin of GH 11640
+        df = pd.DataFrame(columns=['a', 'b', 'c', 'd'],
+                          data=[[1, 'b1', 'c1', 3],
+                                [1, 'b2', 'c2', 4]])
+        dg = df.pivot_table(index='a', columns=['b', 'c'], values='d')
+        dg = dg.reset_index()
```

Comment:
Frankly, I don't have much much of an opinion on the subject. But if `.drop('a', axis=1)` does not work, then `.groupby('a')` will not work, while `.groupby(dg['a']) will work. Then, this part of the documentation could be rewritten


> The mapping can be specified many different ways:
> ...
> - For DataFrame objects, a string indicating a column to be used to group. Of course df.groupby('A') is just syntactic sugar for df.groupby(df['A']), but it makes life simpler